tests/utils/expressions.js

```diff
@@ -0,0 +1,37 @@
+export const getProductNameExpression = ( productTitle ) =>
+	`a[contains(text(), "${ productTitle }")]`;
+
+export const getQtyItemExpression = ( args ) =>
+	'div[@class="wc-block-cart-item__quantity" and div[@class="wc-block-components-quantity-selector" and ' +
+	getQtyInputExpression( args ) +
+	']]';
+
+export const getQtyInputExpression = ( args = {} ) => {
+	let qtyValue = '';
+
+	if ( args.qty ) {
+		qtyValue = ` and @value="${ args.qty }"`;
+	}
+
+	return (
+		'input[contains(@class, "wc-block-components-quantity-selector__input")' +
+		qtyValue +
+		']'
+	);
+};
+
+export const getQtyPlusButtonExpression = () => {
+	return 'button[contains(@class, "wc-block-components-quantity-selector__button--plus")]';
+};
+
+export const getQtyMinusButtonExpression = () => {
+	return 'button[contains(@class, "wc-block-components-quantity-selector__button--minus")]';
+};
+
+export const getCartItemExpression = ( productTitle, args ) =>
+	'//tr[contains(@class, "wc-block-cart-items__row") and ' +
+	'td[@class="wc-block-cart-item__product" and div[@class="wc-block-cart-item__wrap" and ' +
```

Comment:
What do you think about making these selectors a little less specific? We should be able to refactor the html here to not use tables, without breaking this test.

Reply:
Removed table elements from the query, and kept only: `//div[@class="wc-block-cart-item__wrap"...`